client/state/reader/follows/actions.js

```diff
@@ -47,3 +52,52 @@ export function recordUnfollow( url ) {
 		} );
 	};
 }
+
+/**
+ * Returns an action object to signal that followed sites have been received.
+ *
+ * @param  {Array}  follows Follows received
+ * @return {Object} 		Action object
+ */
+export function receiveFollows( follows ) {
```

Comment:
success is usually used to indicate that the api request completed, where receive is used to accept data that came from the request. Just a convention that's been used in other parts of the state tree.